e2e-tests/src/meroctl.rs

```diff
@@ -0,0 +1,167 @@
+            .expect("data.contextId not found");
+        let member_public_key = json["data"]["memberPublicKey"]
+            .as_str()
+            .expect("data.memberPublicKey not found");
+
+        Ok((context_id.to_owned(), member_public_key.to_owned()))
+    }
+
+    pub async fn context_invite(
+        &self,
+        node_name: &str,
+        context_id: &str,
+        inviteer_public_key: &str,
+        invitee_public_key: &str,
+    ) -> EyreResult<String> {
+        let json = self
+            .run_cmd(
+                node_name,
+                Box::new([
+                    "context",
+                    "invite",
+                    context_id,
+                    inviteer_public_key,
+                    invitee_public_key,
+                ]),
+            )
+            .await?;
+
+        let data = json["data"]
+            .as_str()
+            .expect("Invite response data not found");
+
+        Ok(data.to_owned())
+    }
+
+    pub async fn context_join(
+        &self,
+        node_name: &str,
+        private_key: &str,
+        invitation_data: &str,
+    ) -> EyreResult<(String, String)> {
+        let json = self
+            .run_cmd(
+                node_name,
+                Box::new(["context", "join", private_key, invitation_data]),
+            )
+            .await?;
+
+        let context_id = json["data"]["contextId"]
+            .as_str()
+            .expect("data.contextId not found");
+        let member_public_key = json["data"]["memberPublicKey"]
+            .as_str()
+            .expect("data.memberPublicKey not found");
+
+        Ok((context_id.to_owned(), member_public_key.to_owned()))
+    }
+
+    pub async fn identity_generate(&self, node_name: &str) -> EyreResult<(String, String)> {
+        let json = self
+            .run_cmd(node_name, Box::new(["identity", "generate"]))
+            .await?;
+
+        let public_key = json["data"]["publicKey"]
+            .as_str()
+            .expect("data.publicKey not found");
+        let private_key = json["data"]["privateKey"]
+            .as_str()
+            .expect("data.privateKey not found");
+
+        Ok((public_key.to_owned(), private_key.to_owned()))
+    }
+
+    pub async fn json_rpc_execute(
+        &self,
+        node_name: &str,
+        context_id: &str,
+        method_name: &str,
+        args_json: &serde_json::Value,
+    ) -> EyreResult<serde_json::Value> {
+        let args_json = serde_json::to_string(args_json)?;
+        let json = self
+            .run_cmd(
+                node_name,
+                Box::new([
+                    "json-rpc",
+                    context_id,
+                    method_name,
+                    "--args-json",
+                    &args_json,
+                ]),
+            )
+            .await?;
+
+        println!("{:?}", json);
+
+        Ok(json)
+    }
+
+    async fn run_cmd(&self, node_name: &str, args: Box<[&str]>) -> EyreResult<serde_json::Value> {
```

Comment:
```suggestion
    async fn run_cmd(&self, node_name: &str, args: &[&str]) -> EyreResult<serde_json::Value> {
```

and others, since `run_cmd` only extends another vec with these